src/clojure/nrepl/middleware/interruptible_eval.clj

```diff
@@ -72,7 +72,7 @@
 
    It is assumed that `bindings` already contains useful/appropriate entries
    for all vars indicated by `clojure.main/with-bindings`."
-  [bindings {:keys [code ns transport session eval file line column] :as msg}]
+  [bindings {:keys [code ns transport session eval file line column trim] :as msg}]
```

Comment:
I'd probably call this `truncate`, as `trim` usually carries a different meaning.